crates/forge/src/cheatcodes_hint_processor.rs

```diff
@@ -224,7 +224,42 @@ impl CairoHintProcessor<'_> {
                 self.cheatnet_state.stop_prank(contract_address);
                 Ok(())
             }
-            "mock_call" => todo!(),
+            "start_mock_call" => {
+                let contract_address = ContractAddress(PatriciaKey::try_from(StarkFelt::new(
+                    inputs[0].clone().to_be_bytes(),
+                )?)?);
+                let function_name = inputs[1].clone();
+                let function_name = as_cairo_short_string(&function_name).unwrap_or_else(|| {
+                    panic!("Failed to convert {function_name:?} to Cairo short str")
+                });
+                let function_name = selector_from_name(function_name.as_str());
+
+                let ret_data_length = inputs[2]
+                    .to_usize()
+                    .expect("Missing ret_data len in inputs");
+                let mut ret_data = vec![];
+                for felt in inputs.iter().skip(3).take(ret_data_length) {
+                    ret_data.push(felt_to_stark_felt(&felt.clone()));
+                }
```

Comment:
```suggestion
                let ret_data = Vec::from(&inputs[3..(3 + ret_data_length)]);
```

Reply:
We will still need to convert it into `StarkFelt`

Reply:
True, can map tho

Reply:
yep, but is there any benefit of using `.into_iter().map(...).collect()` instead of the current solution?

Reply:
More of a nit, I don't mind